src/tools/rados/rados.cc

```diff
@@ -903,7 +904,10 @@ class RadosBencher : public ObjBencher {
   }
 
   int aio_remove(const std::string& oid, int slot) {
-    return io_ctx.aio_remove(oid, completions[slot]);
+    if (forcefull)
+      return io_ctx.aio_remove(oid, completions[slot], CEPH_OSD_FLAG_FULL_FORCE);
+    else
+      return io_ctx.aio_remove(oid, completions[slot]);
```

Comment:
Maybe forceflag state should set an int flags and then just have one call to aio_remove:

<pre>
return io_ctx::aoi_remove(oid, completes[slot], flags);
</pre>